src/normalize.js

```diff
@@ -77,16 +77,19 @@ const normalizeGroup = (field, value, locale, entries, createNodeId) => {
 };
 
 const normalizeModularBlock = (blocks, value, locale, entries, createNodeId) => {
-    let modularBlocksObj = [];
+    const modularBlocksArray = [];
+    if (!Array.isArray(value)) return modularBlocksArray;
```

Comment:
here we need to check value is an array before attempting to map over it. otherwise it throws `cannot read property map of value` or similar